samples/ControlCatalog/MainWindow.xaml.cs

```diff
@@ -63,8 +63,18 @@ private void InitializeComponent()
             // TODO: iOS does not support dynamically loading assemblies
             // so we must refer to this resource DLL statically. For
             // now I am doing that here. But we need a better solution!!
-            var theme = new Avalonia.Themes.Default.DefaultTheme();
-            theme.TryGetResource("Button", out _);
+            // Note, theme swiching probably will not work in runtime for iOS.
+            if (Application.Current.Styles.Contains(App.FluentDark)
+                || Application.Current.Styles.Contains(App.FluentLight))
```

Comment:
Well, I don't like it.
Is it still actual or we can remove those lines at all?

Reply:
this is required for ios unfortunately... basically we need to create a reference to some type in the theme assemblies... might be a nicer way?

Reply:
Maybe the XAML compiler can generate these for AOT scenarios?

Reply:
@kekekeks can XamlX solve that issue?